src/decider/agent/copyrightDeactivationClutterRemovalScript.py

```diff
@@ -66,14 +66,6 @@ def copyrightDeactivationMain(df, clutter_flag):
         list_of_copyrights = list_of_copyrights.replace(
             cp_symbol, "copyright")
 
-    if substring in list_of_copyrights:
-      list_of_copyrights = list_of_copyrights.replace(
-          substring, "copyright")
-
-    if cp_symbol in list_of_copyrights:
-      list_of_copyrights = list_of_copyrights.replace(
-          cp_symbol, "copyright")
-
```

Comment:
Yes, but it should also cover all the edge cases which were handled by the previous code.
Also, I would recommend creating a dictionary of `substring` and `cp_symbol` and a loop over it for replacement.
This will allow us to easily add more patterns and replacements in the future if needed.